frame/dvm/rpc/runtime-api/src/lib.rs

```diff
@@ -86,24 +86,24 @@ sp_api::decl_runtime_apis! {
 			estimate: bool,
 		) -> Result<dp_evm::CreateInfo, sp_runtime::DispatchError>;
 		/// Return the current block.
-		fn current_block() -> Option<EthereumBlock>;
+		fn current_block() -> Option<EthereumBlockV0>;
```

Comment:
Will types.json need to updated?

Reply:
Not sure.
We had never maintained this.

Reply:
cc @sxlwar @chunqiuyiyu  , we might need to check this using Pangolin/Pangoro apps.

Reply:
![image](https://user-images.githubusercontent.com/11801722/144971729-3f90bb9a-b0ca-4871-ad19-ed62a898960e.png)

I tested this in my local env just now. It works well.